pre_commit_hooks/loaderon_hooks/general_hooks/check_location.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+"""Checks files locations."""
+import os
+import re
+
+from pre_commit_hooks.loaderon_hooks.util.string_helpers import matches_any_regexp
+from pre_commit_hooks.loaderon_hooks.util.template_methods.file_checker_template_method import FileCheckerTemplateMethod
+
+
+class LocationChecker(FileCheckerTemplateMethod):
+    def _add_arguments_to_parser(self):
+        super(LocationChecker, self)._add_arguments_to_parser()
+        self.parser.add_argument(
+            '-ed', '--directories', action='append',
+            help='Directory regex to be added to white list. Can be set multiple times to add'
+                 'multiple directories to white list.',
+        )
+        self.parser.add_argument(
+            '-ef', '--files', action='append',
```

Comment:
generally it's an antipattern to have multi-letter short arguments (since they don't really fit well into POSIX)